src/main/scala/com/typesafe/sbt/packager/docker/Keys.scala

```diff
@@ -37,7 +37,7 @@ trait DockerKeys {
     "Docker CMD. Used together with dockerEntrypoint. Arguments passed in exec form"
   )
   val dockerExecCommand = SettingKey[Seq[String]]("dockerExecCommand", "The shell command used to exec Docker")
-  val dockerVersion = TaskKey[Option[DockerVersion]]("dockerVersion", "The docker server version")
+  val dockerApiVersion = TaskKey[Option[DockerApiVersion]]("dockerApiVersion", "The docker server version")
```

Comment:
Unfortunately we can not do this in minor or patch releases as this breaks every downstream client.
We would have to 
- introduce a new setting `dockerApiVersion`
- deprecate the old setting
- introduce a new setting to switch between "server version" and "api version"  compatibility mode